LocalSettings.php

```diff
@@ -4040,6 +4040,12 @@
 	'wgPageTriageInfinitScrolling' => array(
 		'default' => true,
 	),
+	
+	// ParserFunctions
+	'wgPFEnableStringFunctions' => array(
```

Comment:
This is already defined here https://github.com/miraheze/mw-config/blob/20397f36409a9151ec27ff9cf6ee2225628ffb0a/LocalSettings.php#L1263

Reply:
Ibidem!